specs/protocol/ecotone/l1-attributes.md

```diff
@@ -38,8 +38,8 @@ The overall calldata layout is as follows:
 | l1BlockHash       | bytes32 | 100-131        | 4       |
 | batcherHash       | bytes32 | 132-163        | 5       |
 
-Total calldata length MUST be exactly 164 bytes, implying the sixth and final segment is only
```

Comment:
This may have been applicable to an earlier version of the ecotone calldata, but it was updated to only have 5 segments, and they are all full.